analyses/cms-open-data-ttbar/analysis.py

```diff
@@ -6,7 +6,7 @@
 
 import ROOT
 from distributed import Client, LocalCluster, SSHCluster, get_worker
-from plotting import save_plots
+from plotting import save_plots, save_ml_plots
```

Comment:
this file also needs a pass of formatting with black :)

Reply:
done